task/distributed_queue.py

```diff
@@ -82,7 +87,7 @@ def __init__(
         any extra arguments in **kwargs will be passed to queue_declare()
 
         the results of the result declarations are stored in
-        DistributedQueue.declare_results, a dict mapping queue name to result
+        DistributedQueue.declare_results, a dict mapping queue name to DeclareResult
```

Comment:
The fact that `DeclareResult` only appears in comments in this module strikes me as a bit strange.  It probably makes sense to annotate `declare_results` as something better than `dict[str, Any]`...

Reply:
Good point! Done.